storage/local/storage.go

```diff
@@ -291,12 +277,21 @@ func NewMemorySeriesStorage(o *MemorySeriesStorageOptions) *MemorySeriesStorage
 			},
 			[]string{discardReasonLabel},
 		),
-		nonExistentSeriesMatchesCount: prometheus.NewCounter(prometheus.CounterOpts{
+		nonExistentSeriesMatches: prometheus.NewCounter(prometheus.CounterOpts{
 			Namespace: namespace,
 			Subsystem: subsystem,
 			Name:      "non_existent_series_matches_total",
 			Help:      "How often a non-existent series was referred to during label matching or chunk preloading. This is an indication of outdated label indexes.",
 		}),
+		memChunks: prometheus.NewGaugeFunc(
+			prometheus.GaugeOpts{
+				Namespace: namespace,
+				Subsystem: subsystem,
+				Name:      "memory_chunks",
+				Help:      "The current number of chunks in memory, excluding cloned chunks (i.e. chunks without a descriptor).",
```

Comment:
Actually not a change of this PR (just moved stuff around), but your suggestion makes sense. Will change.